packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -124,7 +124,7 @@ const TESTS = new Map([
       { test: 'include', expected: configUrl('subscriptionTermsUrl', 'new-subscription', 'subscription-terms') },
       { test: 'include', expected: configUrl('subscriptionSupportUrl', 'new-subscription', 'subscription-support') },
       { test: 'include', expected: `Welcome to ${subscriptionProductEmail}!` },
-      { test: 'include', expected: `If you haven\'t already downloaded ${subscriptionProductEmail}, let\'s get started using all the features included in your subscription:` },
+      { test: 'include', expected: `If you haven't already downloaded ${subscriptionProductEmail}, let's get started using all the features included in your subscription:` },
```

Comment:
Have you verified that removing the `\` chars doesn't mess up the rendering of the email template? I'm not sure how to test rendering this email, and this change makes me a bit nervous.

Reply:
I have not, these are test emails. Just removed because of the [no-useless-escape](https://eslint.org/docs/rules/no-useless-escape) rule.